cmd/attachservice.go

```diff
@@ -33,7 +33,10 @@ var attachServiceCmd = &cobra.Command{
 
 		logging.Info("Attaching service", logging.NewPair("service-name", serviceName))
 
-		httpClient := adapters.NewApiClient(Config.VampCloudHost, Config.VampCloudBasePath, ApiVersion, Config.APIKey)
+		httpClient, err := adapters.NewApiClient(Config.VampCloudAddr, ApiVersion, Config.APIKey)
+		if err != nil {
+			return err
```

Comment:
Should we add context to the error here as well? (there are more places in other files too)

Reply:
I think if we add it in the adapter there's no need to add it here as well. After adding to the adapter the error will already state that creating the api client failed

Reply:
ok

Reply:
I added some errors wrapping where it looked like it would be useful